org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,462 @@
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.12.3",
+                    "url": "git://anongit.freedesktop.org/gstreamer/gst-python",
+                    "commit": "705c1c09228c0a95150ea86cd536bcb0c2c74e51"
+                }
+            ]
+        },
+        {
+            "name": "gst-devtools",
+            "config-opts": [],
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": ["-Ddisable_gtkdoc=true", "--libdir=lib"],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.12.3",
+                    "url": "git://anongit.freedesktop.org/gstreamer/gst-devtools",
+                    "commit": "4a95cb6fb519e6d5ef2f5b0e33a4205da987059d"
+                }
+            ]
+        },
+        {
+            "name": "gstreamer-vaapi",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": ["-Ddisable_gtkdoc=true", "--libdir=lib"],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.12.3",
+                    "url": "git://anongit.freedesktop.org/gstreamer/gstreamer-vaapi",
+                    "commit": "8d2884ec349ba8e402e9251c6e96b2e1ff23da69"
+                }
+            ]
+        },
+        {
+            "name": "gst-editing-services",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": ["-Ddisable_gtkdoc=true", "--libdir=lib"],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.12.3",
+                    "url": "git://anongit.freedesktop.org/gstreamer/gst-editing-services",
+                    "commit": "1ef0ba86926e2b186548ae22353bd7bbf9e3bd77"
+                },
+                {
+                    "type": "patch",
+                    "path": "ges/0001-video-transition-Make-use-of-the-new-compositor-cros.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "ges/0002-transition-Fix-usage-of-transition-types-crossfade.patch"
+                }
+            ]
+        },
+        {
+            "name": "gst-transcoder",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": ["-Ddisable_doc=true", "--libdir=lib"],
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.12.2",
+                    "commit": "5caeea2b0576a7e5d5463377f4c1e6d2605feab1",
+                    "url": "https://github.com/pitivi/gst-transcoder"
+                }
+            ]
+        },
+        {
+            "name": "pitivi",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": ["--libdir=lib"],
```

Comment:
You can drop all of these `--libdir=lib` now, this was patched in the Sdk.

Reply:
There is only here, without passing it explicitly we get issues, see https://phabricator.freedesktop.org/rPTVf5b106eec08f35d29d33ab75db0f5fcbe755f915